src/ecs/rendering/bundle.rs

```diff
@@ -0,0 +1,127 @@
+//! ECS rendering bundle
+
+use renderer::Config as DisplayConfig;
+use renderer::Rgba;
+use renderer::prelude::*;
+
+use app::ApplicationBuilder;
+use assets::{BoxedErr, Loader, AssetFuture};
+use ecs::ECSBundle;
+use ecs::rendering::components::*;
+use ecs::rendering::resources::{Factory, AmbientColor};
+use ecs::rendering::systems::RenderSystem;
+use ecs::transform::components::*;
+use error::{Error, Result};
+
+/// Rendering bundle
+///
+/// Will register all necessary components needed for rendering, along with any resources.
+/// Will also register asset contexts with the asset Loader, and add systems for merging
+/// AssetFutures into their respective component.
+///
+/// Will add RenderSystem as a thread local system.
```

Comment:
Should be `RenderSystem`

Reply:
That's what he wrote?

Reply:
Maybe he means absence of grave acent "`"?